code/__DEFINES/power/apc_defines.dm

```diff
@@ -14,9 +14,9 @@
 /// The APCs cover is missing.
 #define APC_COVER_REMOVED 2
 
-// !NOT IMPLEMENTED! APC visuals 
-/// Pixel offset of the APC from the floor turf 
-//#define APC_PIXEL_OFFSET 25
+// APC visuals
+/// Pixel offset of the APC from the floor turf
+#define APC_PIXEL_OFFSET 25
```

Comment:
Not a power of 2, round it off to 24 possibly or 26

Reply:
Is this meant to be a suggestion comment

Reply:
24 and 26 are even numbers, not powers of 2. Why would it matter anyway?

Reply:
I dont think it being even matters. If it looks fine in my opinion, I think thats enough. I did a bunch of manual edits to fix the screwups of the script, I think its in a good place.

If you see any glaring issues where wallitems dirs or pixelshifts look bad through MDB, sure I can fix it but Im not going to act on arbitrary numbers.